Makefile

```diff
@@ -15,3 +17,18 @@ lint:
 
 coverage:
 	@poetry run pytest --cov shared_memory_dict/ --cov-report=term-missing --cov-report=xml
+
+release:
+	@$(eval VERSION := $(shell poetry version $(RELEASE_TYPE) | cut -d' ' -f6-))
+	@git add .
+	@git commit -m "Bump a $(RELEASE_TYPE) release to version $(VERSION)"
+	@git tag $(VERSION)
```

Comment:
Não quis colocar o bumpversion?

Reply:
no (for now) 'cause I'm thinking isn't necessary

Reply:
(and I'm not sure if is security to change pyproject.toml with bumpversion on every bump)